dateutil/test/test_isoparser.py

```diff
@@ -283,7 +321,6 @@ def test_iso_raises_failing(isostr, exception):
         isoparse(isostr)
 
 
-###
```

Comment:
Don't delete these "header" comments.

Reply:
The pep8-compliant way to do this I've seen elsewhere is `# --------------------------------------`